app/grandchallenge/reader_studies/urls.py

```diff
@@ -76,14 +76,14 @@
         name="ground-truth-remove",
     ),
     path(
-        "<slug:slug>/ground-truth/add/",
-        AddGroundTruthToReaderStudy.as_view(),
-        name="add-ground-truth",
+        "<slug:slug>/ground-truth/add/csv/",
+        AddGroundTruthViaCSVToReaderStudy.as_view(),
+        name="add-ground-truth-csv",
     ),
     path(
-        "<slug:slug>/ground-truth/example/",
-        ReaderStudyExampleGroundTruth.as_view(),
-        name="example-ground-truth",
+        "<slug:slug>/ground-truth/add/csv/example/",
```

Comment:
add/csv/example is misleading. Maybe these should be `<slug:slug>/ground-truth/csv/create/` and `<slug:slug>/ground-truth/csv/example/`?